src/libtelio.udl

```diff
@@ -556,10 +556,18 @@ dictionary FeatureFirewall {
     boolean boringtun_reset_conns;
 };
 
+/// Link detection enhanced detection config
+dictionary FeatureEnhancedDetection {
+    /// Number of pings issued during the connection check
+    u32 no_of_pings;
+};
+
 /// Link detection mechanism
 dictionary FeatureLinkDetection {
     /// Configurable rtt in seconds
     u64 rtt_seconds;
+    /// Check the link state before reporting it as down
+    FeatureEnhancedDetection? enhanced_detection;
```

Comment:
I think it might be worth just inlining the number of pings in here. That would reduce the code and should be a very simple modification, now we have a nested object one feature each and to be fair, nobody should use link detection without at least one ping(enhancement) :)

Reply:
Yes, but as you said we might add some interval config. 
And if I inline it as a number how would someone disable the feature? Making it 0?